easybuild/easyconfigs/b/Biggus/Biggus-0.13.0-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'Biggus'
+version = '0.13.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/SciTools/biggus/'
+description = """Virtual large arrays and lazy evaluation."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['https://github.com/SciTools/Biggus/archive/']
```

Comment:
Any reason for preferring this over PyPI?

That's usually the better option, since sometimes tarballs available from PyPI are pre-processed some way and not exactly the same as the ones available from GitHub...